consensus/istanbul/types.go

```diff
@@ -26,6 +26,15 @@ import (
 	"github.com/ethereum/go-ethereum/rlp"
 )
 
+type Uptime struct {
+	Score           uint64
```

Comment:
nit: Consider something else (e.g. `Count`), since `Score` is an overloaded word. We use `Score` to refer to the exponential moving average